### ISSUE TITLE
Helpers resolving data-object

### ISSUE THREAD

**Issue by SamuelRott** [zordius/lightncandy#253]
The PHP Code:
-------------
```
// Helpers:
$helpers = array(
  'foo' => function () {
    return 'Hello World';
  },
);

// Input Data :
 {
    foo {
       title: "bar"
    }
}

```

The Issue:
----------
if you have an helper called `foo` and a data-object called `foo` lightncandy will try to resolve `{{foo.title}}` as a helper and not a string.  

Not sure its an issue.
The good part of this behaviour is that you need to be more precise/specific in your naming. 
The bad part is that its not the same behaviour as handlebars.

**Comment by zordius** [comment]
Example jsfiddle created: https://jsfiddle.net/4c3y1dny/

**Comment by zordius** [comment]
It should be fixed in current master branch, please try again, thanks.

**Comment by zordius** [comment]
released in 1.0.1
